test/unit/worker.js

```diff
@@ -70,9 +80,7 @@ describe('Worker', function () {
     beforeEach(function () {
       opts.runNow = false;
       worker = Worker.create(opts);
-      sinon.stub(error, 'log');
     });
-    afterEach(function () { error.log.restore(); });
```

Comment:
Addressed this, we now ensure we are logging and reporting errors during `Worker.prorotype.run`